### PR TITLE
Use eredis from Hex and bump version

### PR DESCRIPTION
**Pull request by zuiderkwast**
Generated docs also updated.

### CODE REVIEW

File: doc/eredis_cluster.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ redis_result() = <a href="#type-redis_simple_result">redis_simple_result()</a> |
 
 
 <pre><code>
-redis_simple_command() = [<a href="#type-anystring">anystring()</a>]
+redis_simple_command() = [<a href="#type-anystring">anystring()</a> | integer()]
```
Comment (bjosv) [inline doc/eredis_cluster.md:125]: when is the command and integer?

Comment (zuiderkwast) [reply doc/eredis_cluster.md:125]: In scan/4, the spec says some args are integers. The full command is a list. The first element is never an integer, but args may be, for convenience and are sent as strings. The encoder actually accepts more types but integer is the only arg type which makes sense.

**Comment by bjosv** [comment]
LGTM

**Comment by zuiderkwast** [comment]
OK. Let's wait until we have solved the pool size config not working in connect/2 or at least made a note about it in the docs.

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-0.5.12
+0.6.0-nordix
```
Comment (alexandremcosta) [inline VERSION:1]: Can you also tag it to make it easier to control version using mix `tag: "..."`?

Comment (zuiderkwast) [reply VERSION:1]: Of course! We'll tag it when this is merged. I though this was already merged and tagged, but it was forgotten.

**Comment by bjosv** [comment]
> OK. Let's wait until we have solved the pool size config not working in connect/2 or at least made a note about it in the docs.

This was corrected by https://github.com/Nordix/eredis_cluster/pull/42 wasn't it

File: rebar.config [hunk 3]
```diff
@@ -39,7 +39,8 @@
             , {preprocess, true}
             ]}.
 
-{deps, [{eredis, {git, "git://github.com/Nordix/eredis.git", {ref, "a83413a"}}},
+{deps, [{eredis, {git, "git://github.com/Nordix/eredis.git",
+                  {tag, "v1.3.0-nordix"}}},
```
Comment (alexandremcosta) [inline rebar.config:43]: Shouldn't be `v1.3.1-nordix` ?

File: VERSION (hunk 2 above)
Comment (alexandremcosta) [reply VERSION:1]: Hey guys, I believe we can call it just 0.6.0 from now on. I'm confident we will get `eredis_cluster` ownership and I doubt there will be any new versions from adrienmo/eredis_cluster.

File: rebar.config (hunk 3 above)
Comment (zuiderkwast) [reply rebar.config:43]: Yes, you're right. Should we wait a bit until it's actually official?

**Review by alexandremcosta — APPROVED** [review]
If we fear any backwards incompatibility, we should bump to 1.0.0

File: rebar.config (hunk 3 above)
Comment (alexandremcosta) [reply rebar.config:43]: Now we should depend on 1.3.2 on hex 😅

**Comment by zuiderkwast** [comment]
> If we fear any backwards incompatibility, we should bump to 1.0.0

It's OK to make backwards incompatible changes between beta versions, so 0.6.0 should be just as fine.

Of course, we should try to keep it as much BC as possible, but maybe some handling of ASK-redirects and such things can be changed still before we reach 1.0.0.

File: VERSION (hunk 2 above)
Comment (zuiderkwast) [reply VERSION]: I don't know what this VERSION file is for, but it's updated to 6.0.0 now.

**Comment by zuiderkwast** [comment]
This and #48 are independent, I think. Shall we merge both, then tag 6.0.0, then publish to Hex?

**Review by bjosv — APPROVED** [review]

**Comment by bjosv** [comment]
> This and #48 are independent, I think. Shall we merge both, then tag 6.0.0, then publish to Hex?

Sound good, lets get a 0.6.0 ready

**Pull request merged**